<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CypressMainWindow</class>
 <widget class="QMainWindow" name="CypressMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CLSA Cypress</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="testParticipant">
      <property name="title">
       <string>Participant</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="barcode">
         <item>
          <widget class="QLabel" name="barcodeLabel">
           <property name="text">
            <string>Barcode</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignRight">
          <widget class="QLineEdit" name="barcodeValue">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="weight">
         <item>
          <widget class="QLabel" name="weightLabel">
           <property name="text">
            <string>Weight </string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignRight">
          <widget class="QLineEdit" name="weightValue">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="height">
         <item>
          <widget class="QLabel" name="heightLabel">
           <property name="text">
            <string>Height  </string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignRight">
          <widget class="QLineEdit" name="heightValue">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="dateOfBirth">
         <item>
          <widget class="QLabel" name="dateOfBirthLabel">
           <property name="text">
            <string>Date of Birth</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDateEdit" name="dateOfBirthValue">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="sex">
         <item>
          <widget class="QLabel" name="sexLabel">
           <property name="text">
            <string>Sex        </string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="sexValue">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>Female</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Male</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="stages">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Stages</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="measureRoom1">
         <property name="title">
          <string>Measure Room 1</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="launchWeightScale">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Weight</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="launchBpm">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>BPM</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="launchEcg">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>ECG</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="launchDxa1">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>DXA (Hip)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="launchDxa2">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>DXA (WB)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="launchUltrasound">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Ultrasound</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="launchSpirometer">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Spirometer</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="interviewRoom1">
         <property name="title">
          <string>Interview Room 1</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="launchAudiometer">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Audiometer</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="launchCDTT">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>CDTT</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="launchChoiceReaction">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Choice Reaction</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="interviewRoom2">
         <property name="title">
          <string>Interview Room 2</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="launchRetinalCamera">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Retinal Camera</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="launchTonometer">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Tonometer</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="launchGripStrength">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Grip Strength</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="launchFrax">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>FRAX</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCIMT">
   <property name="text">
    <string>CIMT</string>
   </property>
  </action>
  <action name="actionECG">
   <property name="text">
    <string>ECG</string>
   </property>
  </action>
  <action name="actionBPM">
   <property name="text">
    <string>BPM</string>
   </property>
  </action>
  <action name="actionAudiometer">
   <property name="text">
    <string>Audiometer</string>
   </property>
  </action>
  <action name="actionCDTT">
   <property name="text">
    <string>CDTT</string>
   </property>
  </action>
  <action name="actionAudiometer_2">
   <property name="text">
    <string>Audiometer</string>
   </property>
  </action>
  <action name="actionCDTT_2">
   <property name="text">
    <string>CDTT</string>
   </property>
  </action>
  <action name="actionCCB">
   <property name="text">
    <string>CCB</string>
   </property>
  </action>
  <action name="actionGrip_Strength">
   <property name="text">
    <string>Grip Strength </string>
   </property>
  </action>
  <action name="actionVividI">
   <property name="text">
    <string>GE VividI </string>
   </property>
  </action>
  <action name="actionECG_2">
   <property name="text">
    <string>ECG (MAC800)</string>
   </property>
  </action>
  <action name="actionDEXA">
   <property name="text">
    <string>DEXA</string>
   </property>
  </action>
  <action name="actionBody_Comp">
   <property name="text">
    <string>Body Comp</string>
   </property>
  </action>
  <action name="actionFrax">
   <property name="text">
    <string>Frax</string>
   </property>
  </action>
  <action name="actionRetinal_Cam">
   <property name="text">
    <string>Retinal Cam</string>
   </property>
  </action>
  <action name="actionSignature">
   <property name="text">
    <string>Signature</string>
   </property>
  </action>
  <action name="actionTonometer">
   <property name="text">
    <string>Tonometer</string>
   </property>
  </action>
  <action name="actionWeigh_Scale">
   <property name="text">
    <string>Weigh Scale</string>
   </property>
  </action>
  <action name="actionSpirometer">
   <property name="text">
    <string>Spirometer</string>
   </property>
  </action>
  <action name="actionBlood_Pressure">
   <property name="text">
    <string>Blood Pressure</string>
   </property>
  </action>
  <action name="actionTest_Locally">
   <property name="text">
    <string>Test Locally</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
